guides/v2.1/cloud/project/new-relic.md

```diff
@@ -1,44 +1,34 @@
 ---
 group: cloud-guide
-title: New Relic APM
+title: New Relic services
 functional_areas:
   - Cloud
   - Integration
   - Setup
   - Services
 ---
 
-[New Relic for application performance management (APM)](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-new-relic-apm) is a software analytics product that helps you analyze and improve application interactions. Each {{site.data.var.ece}} plan includes a New Relic APM license that supports up to three instances across all environments. 
-
-New Relic APM provides the following features to Magento:
+[New Relic Infrastructure](https://newrelic.com/products/infrastructure) automatically connects with your application data and performance analytics to provide dynamic server monitoring and is standard for {{site.data.var.ece}} accounts. [New Relic for application performance management (APM)](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-new-relic-apm) is a software analytics product that helps you analyze and improve application interactions. New Relic APM provides the following features to Magento:
```

Comment:
Noticed APM in the last sentence. Should it be removed?

Reply:
Heya! <wave>
Actually, APM is still a thing. It is just not the ONLY thing. :) I might receive MORE information about this soon, so we shall see!

Reply:
Do we need to mention that Infra is for Pro Production only?

Reply:
I've seen Infrastructure available on Pro staging as well.

Reply:
As discussed in Slack - that looks to be a provisioning bug. Working with infra team remove it from all stagings if it's there.